modules/nlu/src/backend/pipelines/slots/pre-processor.ts

```diff
@@ -96,7 +92,7 @@ export const generateTrainingSequence = (langProvider: LanguageProvider) => asyn
   let tokens: Token[] = []
   let matches: RegExpExecArray | null
   const genToken = _generateTrainingTokens(langProvider)
-  const cannonical = _removeEntityNotations(input)
+  const cannonical = keepEntityValues(input)
```

Comment:
what was before and what is now's result of this?  I think we want to keep it as it was before but merge the `"[", "ENTITY_NAME", "]"` tokens into one token

Reply:
Yeah there's a question whether we should use canonical or slot-type, I think canonical makes sense for now